priv/repo/migrations/20161019090945_add_stripe_customers_cards_tables.exs

```diff
@@ -2,39 +2,37 @@ defmodule CodeCorps.Repo.Migrations.AddStripeCustomersCardsTables do
   use Ecto.Migration
 
   def change do
-    create table(:stripe_customers, primary_key: false) do
-      add :id, :string, primary_key: true
+    create table(:stripe_customers) do
       add :created, :datetime
-      add :currency, :string, null: false, default: "usd"
-      add :delinquent, :boolean, null: false, default: false
-      add :email, :string, null: false
-      add :user_id, references(:users)
+      add :currency, :string
+      add :delinquent, :boolean
+      add :email, :string
+      add :id_from_stripe, :string, null: false
+
+      add :user_id, references(:users, on_delete: :nothing), null: false
```

Comment:
Changed the id, set only pieces that should be null as null, and added a user relationship.